src/topology/continuous_function/algebra.lean

```diff
@@ -340,6 +364,14 @@ instance module : module R C(α, M) :=
 
 variables (R)
 
+/-- Composition on the left by a continuous linear map, as a `linear_map`.
+Similar to `linear_map.comp_left`. -/
+@[simps]
+def _root_.continuous_linear_map.comp_left (α : Type*) [topological_space α] (g : M →L[R] M₂) :
+  C(α, M) →ₗ[R] C(α, M₂) :=
```

Comment:
Is the resulting map not continuous?
```suggestion
  C(α, M) →L[R] C(α, M₂) :=
```

Reply:
There's no topology on `C(α, M)` constructed in this file or its predecessors.